hypothesis-python/RELEASE.rst

```diff
@@ -0,0 +1,5 @@
+RELEASE_TYPE: minor
+
+This release make emails strategy a public API
```

Comment:
Best to avoid describing the file location (as that's a private implementation detail), and it would be nice to hyperlink the name.  I'd write this as:

```
This release adds the :func:`~hypothesis.strategies.emails` strategy,
which generates unicode strings representing an email address.

Thanks to Sushobhit for moving this to the public API (:issue:`162`).
```

Reply:
And that's it!  Once the release note is finalised I'll merge and do a release :smile:

Reply:
ok, will change it accordingly. Hopefully after this PR, these kinds of issues might not occur for future PR(s). :)